src/Mapper/DonorMapper.php

```diff
@@ -48,6 +48,19 @@ public function __construct(CollectionInterface $collection, DonorSchema $donorS
     }
 
     /**
+     * Check if a donor with specified key exists in database
+     * @return boolean
+     */
+    public function hasKey(string $key): boolean
```

Comment:
The keyword for the boolean type is actually `bool`. Thats why you get the confusion error message, it interprets `boolean` as the name of a class in the current namespace...